pom.xml

```diff
@@ -49,21 +49,29 @@
 			<url>https://repo.jenkins-ci.org/public/</url>
 		</pluginRepository>
 	</pluginRepositories>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.jenkins.tools.bom</groupId>
+				<artifactId>bom-2.332.x</artifactId>
+				<version>1500.ve4d05cd32975</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
 	<dependencies>
 		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-databind</artifactId>
-			<version>2.13.2</version>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>jackson2-api</artifactId>

```

Comment:
Our general preference in the Jenkins project is to use API plugins rather than embedding libraries (i.e., dynamic rather than static linking). Another benefit of switching to the API plugin is that its version can be managed by the BOM automatically.